tests/runtime/probe

```diff
@@ -10,7 +10,7 @@ TIMEOUT 5
 
 NAME kprobe_target
 RUN bpftrace -v -e 'kprobe:syscalls:sys_exit_nanosleep { printf("SUCCESS '$test' %d\n", pid); exit(); }'
-EXPECT kprobes should not have a target
+EXPECT kprobe provider requires 1 argument
```

Comment:
This error message took me a while to understand - I think it can improved a little
1. "requires 1 argument" sounds to me like it's missing an argument, not that it has too many
1. I didn't know what it meant by "argument" (I was looking for something like a function call)

I'm just providing criticism here - no ideas are coming to me for better wording at the moment :P

Reply:
Yeah, I don't have anything better. I'm hoping the error underlining makes these errors more clear.